components/script/dom/performanceresourcetiming.rs

```diff
@@ -112,24 +112,24 @@ impl PerformanceResourceTiming {
             entry: PerformanceEntry::new_inherited(
                 DOMString::from(url.into_string()),
                 DOMString::from("resource"),
-                resource_timing.start_time as f64,
-                resource_timing.response_end as f64 - resource_timing.start_time as f64,
+                resource_timing.start_time.as_nanos() as f64,
+                resource_timing.response_end.as_nanos() as f64 - resource_timing.start_time.as_nanos() as f64,
```

Comment:
Can this avoid a bit of repetition?

```suggestion
                (resource_timing.response_end - resource_timing.start_time).as_nanos() as f64,
```